examples/flakes/go-mod/devbox.json

```diff
@@ -0,0 +1,11 @@
+{
+  "packages": [
+    "path:ory-cli"
+  ],
+  "shell": {
+    "init_hook": null
+  },
+  "nixpkgs": {
+    "commit": "f80ac848e3d6f0c12c52758c0f25c10c97ca3b62"
```

Comment:
Should we get rid of `nixpkgs` field?

Reply:
Once we have the auto-latest stuff merged, I'll go through and do that.